### PR TITLE
[3.4] [Validator] Add missing french validators translations

### PR DESCRIPTION
**Pull request by hhamon**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | ~
| License       | MIT
| Doc PR        | ~

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Very nice! We'd also include another new translation. See #30946.

**Comment by hhamon** [comment]
Thanks Javier!

I've put the french translation for the compromised password constraint in @xabbuh's PR so that he can update the file accordingly :)

**Review by lyrixx — APPROVED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf [hunk 1]
```diff
@@ -334,6 +334,10 @@
                 <source>This value should be valid JSON.</source>
                 <target>Cette valeur doit être un JSON valide.</target>
             </trans-unit>
+            <trans-unit id="87">
+                <source>This collection should contain only unique elements.</source>
+                <target>Cette collection ne doit contenir que des éléments uniques.</target>
```
Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: des éléments différents ?

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf [hunk 2]
```diff
@@ -334,6 +334,10 @@
                 <source>This value should be valid JSON.</source>
                 <target>Cette valeur doit être un JSON valide.</target>
             </trans-unit>
+            <trans-unit id="87">
```
Comment (xabbuh) [inline src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: This id should be `91` (translations from #28637 #30900 are missing too).

**Review by xabbuh — DISMISSED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf (hunk 1 above)
Comment (hhamon) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: Changed ;)

Comment (greg0ire) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: Ou maybe "Cette collection ne doit pas comporter de doublons", or do we have to not wander too much away when translating?

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf (hunk 2 above)
Comment (hhamon) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: it's `87` everywhere else.

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf (hunk 1 above)
Comment (fabpot) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: Even better!

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf (hunk 2 above)
Comment (hhamon) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: Fixed

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf [hunk 3]
```diff
@@ -334,6 +334,34 @@
...
+                <source>This collection should contain only unique elements.</source>
+                <target>Cette collection ne doit contenir que des éléments différents.</target>
```
Comment (greg0ire) [inline src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]:
```suggestion
                <target>Cette collection ne doit pas comporter de doublons.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf [hunk 4]
```diff
@@ -334,6 +334,34 @@
...
+            </trans-unit>
+            <trans-unit id="88">
+                <source>This value should be positive.</source>
+                <target>Cette valeur doit être strictement positive.</target>
+            </trans-unit>
+            <trans-unit id="89">
+                <source>This value should be either positive or zero.</source>
+                <target>Cette valeur doit être positive ou égale à zéro.</target>
```
Comment (greg0ire) [inline src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]:
```suggestion
                <target>Cette valeur doit être supérieure ou égale à zéro.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf [hunk 5]
```diff
@@ -334,6 +334,34 @@
...
+            </trans-unit>
+            <trans-unit id="90">
+                <source>This value should be negative.</source>
+                <target>Cette valeur doit être strictement négative.</target>
+            </trans-unit>
+            <trans-unit id="91">
+                <source>This value should be either negative or zero.</source>
+                <target>Cette valeur doit être négative ou égale à zéro.</target>
```
Comment (greg0ire) [inline src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]:
```suggestion
                <target>Cette valeur doit être inférieure ou égale à zéro.</target>
```

**Review by greg0ire — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf (hunk 5 above)
Comment (hhamon) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: Bien vu

Comment (hhamon) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: Fixed. Thank you :)

File: src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf (hunk 3 above)
Comment (hhamon) [reply src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf]: Fixed. Thank you :)

**Comment by hhamon** [comment]
I think it's good to go!

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @hhamon.

**Pull request merged**